.github/workflows/wheels-build.yml

```diff
@@ -121,6 +121,7 @@ jobs:
           cat >>"$GITHUB_ENV" <<EOF
           CIBW_BEFORE_BUILD=bash ./tools/build_pgo.sh $PGO_WORK_DIR $PGO_OUT_PATH
           CIBW_ENVIRONMENT=RUSTUP_TOOLCHAIN=stable RUSTFLAGS='-Cprofile-use=$PGO_OUT_PATH -Cllvm-args=-pgo-warn-missing-function'
+          CIBW_ENVIRONMENT_MACOS=MACOSX_DEPLOYMENT_TARGET='10.12' RUSTUP_TOOLCHAIN=stable RUSTFLAGS='-Cprofile-use=$PGO_OUT_PATH -Cllvm-args=-pgo-warn-missing-function'
```

Comment:
Can we set the `MACOSX_DEPLOYMENT_TARGET` variable in the general `CIBW_ENVIRONMENT` var to avoid duplicating the options?  I think it should just be safely ignored on Linux.

Reply:
It actually would only get shared between macOS and windows since we have a separate linux one already. But it should be ignored on all the other platforms. I'll update it in a sec.

Reply:
Oh so we do - in that case I withdraw the concern. I'm so used to it being Windows that has the special case I forgot.

Now you mention it, I remember reorganising the setup and environment here such that Windows no longer needed special handling.